recipes/audio-separator/conda-forge.yml

```diff
@@ -0,0 +1,5 @@
+channel_priority: strict
+channels:
+  sources:
+    - pytorch
```

Comment:
This would create some complications for conda-forge users b/c we do not recommend folks to mix channels and yet this package would only work with another channel in their setup.

Reply:
Thanks for taking a look at this @ocefpaf !

I've pushed a commit removing this, but the reason I had this here is that conda-forge is missing a `win-64` pytorch build and seems to be outdated compared to the pytorch channel (plus, the pytorch channel is what pytorch themselves tell people to use: https://pytorch.org/get-started/locally/)

Would you happen to know of a good example feedstock which requires pytorch & torchaudio as dependencies from conda-forge?

I've been really struggling to figure out the _right_ way to get this working, and every example I've found myself (just using github code search with various combinations of filters trying to find feedstock repos) has been frustratingly varied in approach. Any assistance greatly appreciated, I just want audio-separator to be available to people on conda-forge!

Thank you!

Reply:
Failing on Windows is fine and folks should be able to add the pytorch channel on their setup at their own discretion. Things should work but we, conda-forge, don't recommend it b/c it can mess up new users setups.